halpybot/packages/announcer/announcer.py

```diff
@@ -32,6 +34,14 @@
     "NW": "SE",
 }
 
+platform_shorts = {
+    Platform.ODYSSEY: "PCO",
+    Platform.XBOX: "XB",
+    Platform.PLAYSTATION: "PS",
+    Platform.LEGACY_HORIZONS: "PCH",
+    Platform.LIVE_HORIZONS: "PCL",
+}
```

Comment:
this dict seems fishy to me.
any reason the enum values couldn't be set to these values?

Reply:
Currently, the database refers to platforms by an ID number, which is likely going to be the most stable way to pass from the site to the bot (Thus why the enum uses ints right now). This dict is (at least designed) to allow us to resurrect the concept of short case references (PCOCASE, PSCASE, PCLCASE) instead of having to rely on "(LIVE) PC-HorizonsCASE" with the current not-very-good method. 

However I'm open to the idea that I'm overcomplicating things here.